### ISSUE TITLE
Broken Symbols in COM output

### ISSUE THREAD

**Issue by Express2000** [kalanda/esp8266-sniffer#18]
Tried various baudrates, different boards (Wemos D1 mini and Witty cloud), nothing helped.

![Image 3](https://user-images.githubusercontent.com/36521367/126080579-dfefac54-d7b3-43e0-aa60-6e5419354c83.png)

**Comment by Express2000** [comment]
Found solution: with old version of Espressif 8266 platform (2.6.3) it works without odd symbols.

**Comment by playaspec** [comment]
It looks like someone is using Unicode in their SSID and that terminal can't handle it.

**Comment by Express2000** [comment]
No, it was not unicode issue. SlowNet is my network and it doesn't have non-Latin characters, but it was screwed in the last line.

**Comment by underwoodblog** [comment]
With 3.1.2 still broken symbols.
